server/adaptors/integrations/repository/integration.ts

```diff
@@ -211,7 +211,7 @@ export class Integration {
       );
       try {
         const ndjson = await fs.readFile(sobjPath, { encoding: 'utf-8' });
-        const asJson = '[' + ndjson.replace(/\n/g, ',') + ']';
+        const asJson = '[' + ndjson.trim().replace(/\n/g, ',') + ']';
```

Comment:
This breaks NDJson compliance. The [spec](https://github.com/ndjson/ndjson-spec#32-parsing) says that the two permissible behaviors are to ignore all blank lines, or raise an exception on any blank line. If we want to switch to the former mode of operation, we need to handle all blank lines, not just ones at the start or end of the file.

Reply:
as stated - regardless of the spec it should work identically to how the savedObject import API work - if that works with the empty lines so does this code

Reply:
Where is the saved object import format specified? It's mentioned in #743 that other implementation quirks make this nontrivial. We shouldn't assume they share the same behavior based on one example.